apps/docs/docs/ref/javascript/typescript-support.mdx

```diff
@@ -147,12 +163,17 @@ You can use the following helper types to make the TypeScript support easier to
   </RefSubLayout.Details>
   <RefSubLayout.Examples>
     ```ts ./database.types.ts
```

Comment:
double check this works.
MDX parser is a bit funny when there are no empty lines at start and end of component children